pkgdb2/lib/__init__.py

```diff
@@ -931,13 +941,13 @@ def search_actions(
         count=count)
 
 
-def search_logs(session, package=None, packager=None,
-                from_date=None, page=None,
-                limit=None, count=False):
+def search_logs(session, package=None, namespace=None, packager=None,
```

Comment:
Here, also, namespace comes *after* ``package`` in the signature.. but having it come first seems like it would be more consistent with other functions, imho.